eventstore/dispatcher.go

```diff
@@ -69,6 +73,7 @@ func (d *Dispatcher) Dispatch(event core.Event) error {
 	// Safe to fire off reducers now that event is persisted
 	g, _ := errgroup.WithContext(context.Background())
 	for _, reducer := range d.reducers {
+		reducer := reducer
```

Comment:
This is a fix for the concurrency issue that took some time to figure out!
The below go routine depended on mutating closure. 
Instead of doing this, there's the other usual: make value a param of go-routine.

Reply:
Good catch... those ones are always hard to spot.